pom.xml

```diff
@@ -255,12 +257,20 @@
       <id>repo.jenkins-ci.org</id>
       <url>https://repo.jenkins-ci.org/public/</url>
     </repository>
+    <repository> <!-- TODO FOR DRAFT -->
+      <id>incrementals.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/incrementals/</url>
+    </repository>
   </repositories>
   <pluginRepositories>
     <pluginRepository>
       <id>repo.jenkins-ci.org</id>
       <url>https://repo.jenkins-ci.org/public/</url>
     </pluginRepository>
+    <pluginRepository><!-- TODO FOR DRAFT -->
+      <id>incrementals.jenkins-ci.org</id>
+      <url>https://repo.jenkins-ci.org/incrementals/</url>
+    </pluginRepository>
```

Comment:
```suggestion
```

Reply:
I must say that I wasnt able to compile it locally without these temporary .pom changes, but I assume they will be removed before it will be merged